### ISSUE TITLE
Renaming scene will break the reference in AssetPlacer library. 

### ISSUE THREAD

**Issue by maxime4000** [CookieBadger/assetplacer-docs#14]
If I have a file name SM_Bld_House_Door_01. I would like to rename it House_Door_01. I do it in the editor, reference are update everywhere but not in AssetPlacer.

## Error message :
Asset file not found. Was at: "res://..."

**Comment by CookieBadger** [comment]
Yes, this is the way I implemented it for now. I will try if switching to using UIDs in addition to paths would fix this in a future update.
